tests/runtime/test_grpc_server.py

```diff
@@ -466,18 +468,24 @@ def test_train_primitive_model(
     train_request_class = DataBase.get_class_for_name(
         "SampleTaskSamplePrimitiveModuleTrainRequest"
     )
-    union_list_str_int_dm = caikit.interfaces.common.data_model.UnionListStrIntSource
-    union_list_str_bool_dm = caikit.interfaces.common.data_model.UnionListStrBoolSource
+    union_list_str_dm = (
```

Comment:
what is the use of parentheses for these statements?

Reply:
None, but for some reason `fmt` adds them for me :D

Reply:
Actually there is something I can do!